components/svelte/src/Particles.svelte

```diff
@@ -5,6 +5,8 @@
   import type { ISourceOptions } from "tsparticles-engine";
   import { tsParticles } from "tsparticles-engine";
 
+  let clazz = "";
```

Comment:
Why not a single export like the other properties?

Reply:
I do this because [we can't create a label/variable/function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#reserved_words) with `class` as the name.
So we create a variable named something other than `class` (in our case `clazz` but it could have been something else)
Why not export the `clazz` variable directly? Because when we create our component, we won't have a `class` props but `clazz` props
```html
<Particles
  id="tsparticles"
  clazz="hello word foo bar"
  options="{particlesConfig}"
  on:particlesLoaded="{onParticlesLoaded}"
  particlesInit="{particlesInit}"
/>
```
We could keep `clazz` as a props but I think it breaks the spirit of Svelte where we feel like we are writing vanilla HTML/CSS/JS without a framework. Having this `clazz` would break the vanilla aspect which is the beauty of Svelte.
So to fix that, we `export { clazz as class }` which allows us when we use the component to pass a `class` props and not a `clazz`.
```html
<Particles
  id="tsparticles"
  class="hello word foo bar"
  options="{particlesConfig}"
  on:particlesLoaded="{onParticlesLoaded}"
  particlesInit="{particlesInit}"
/>
```

Reply:
I thought JavaScript had a way to name a variable with a reserved word, but it's not possible. I didn't research it until now, I just thought it was possible. We can use `cssClass` as a variable name instead of `clazz`, I think it's easier to read, using `class` as an export as you suggested.